CHANGES.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Adds an eps to `y_pred` when calculating loss for a `NeuralNetClassifer` (#491)
```

Comment:
I would prefer a more user-centric description of the change, say:

```suggestion
- Improve numerical stability when using `NLLLoss` in `NeuralNetClassifer` (#491)
```

Reply:
Good point.